src/Joonasw.AspNetCore.SecurityHeaders/Csp/Builder/CspBuilder.cs

```diff
@@ -82,9 +82,25 @@ public void ReportViolationsTo(string uri)
             if(uri.Length == 0) throw new ArgumentException("Uri can't be empty", nameof(uri));
 
             _options.ReportUri = uri;
-        }
+		}
+
+		/// <summary>
+		/// Configures the middelware to include the X-Header.
+		/// </summary>
+		public void IncludeXHeader()
+		{
+			_options.IncludeXHeader = true;
+		}
+
+		/// <summary>
+		/// Configures the middelware to exclude the X-Header.
+		/// </summary>
+		public void ExcludeXHeader()
```

Comment:
I would rather have one function that enables the option. That would be in line with the rest of the API surface here, e.g. AllowNone().